azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/Backend.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.management.network;
+
+import com.microsoft.azure.management.network.implementation.BackendAddressPoolInner;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.ChildResource;
+import com.microsoft.azure.management.resources.fluentcore.model.Attachable;
+import com.microsoft.azure.management.resources.fluentcore.model.Settable;
+import com.microsoft.azure.management.resources.fluentcore.model.Wrapper;
+
+/**
+ * An immutable client-side representation of an load balancer's backend address pool.
+ */
+public interface Backend extends
+    Wrapper<BackendAddressPoolInner>,
+    ChildResource {
+
+    /**
+     * Grouping of backend definition stages.
+     */
+    interface DefinitionStages {
+        /**
+         * The first stage of the backend definition.
+         * @param <ParentT> the return type of the final {@link WithAttach#attach()}
+         */
+        interface Blank<ParentT> extends WithAttach<ParentT> {
+        }
+
+        /** The final stage of the backend definition.
+         * <p>
+         * At this stage, any remaining optional settings can be specified, or the definition
+         * can be attached to the parent load balancer definition using {@link WithAttach#attach()}.
+         * @param <ParentT> the return type of {@link WithAttach#attach()}
+         */
+        interface WithAttach<ParentT> extends
+            Attachable.InDefinition<ParentT> {
+        }
+    }
+
+    /** The entirety of a backend definition.
+     * @param <ParentT> the return type of the final {@link DefinitionStages.WithAttach#attach()}
+     */
+    interface Definition<ParentT> extends
+        DefinitionStages.Blank<ParentT>,
+        DefinitionStages.WithAttach<ParentT> {
+    }
+
+    /**
+     * Grouping of backend update stages.
+     */
+    interface UpdateStages {
+    }
+
+    /**
+     * The entirety of a backend update as part of a load balancer update.
+     */
+    interface Update extends
+        Settable<LoadBalancer.Update> {
+    }
+
+    /**
+     * Grouping of backend definition stages applicable as part of a load balancer update.
+     */
+    interface UpdateDefinitionStages {
```

Comment:
`UpdateStages` is there - see above. It's empty because there is nothing today for Backends that's updatable today. There may be in the future, so it's a placeholder.
Note that 1UpdateDefinitionStages` is a different thing. Backend is a child resource, so like we have everywhere else, the Definition interfaces that are applicable during the parent resource Update need to be put into a separate interface to avoid raw type references. We have this pattern across all the child resources.